app/frontend.py

```diff
@@ -24,6 +26,11 @@ def create_app():
     app = Flask(__name__)
     wb = WyzeBridge()
     try:
+        subprocess.Popen(
```

Comment:
I'm not a huge fan of this. Can we start the Python app and the WHEP Proxy in the Dockerfile simultaneously? This could add extra latency to starting a camera stream.

Reply:
This isn't blocking the stream from starting, this starts up when flask starts.